### PR TITLE
Form processing tweaks

### PR DESCRIPTION
**Pull request by czue**
two very small commits

@dannyroberts wait for build

### CODE REVIEW

File: corehq/ex-submodules/couchforms/util.py [hunk 1]
```diff
@@ -505,7 +506,11 @@ def process(xform):
                             return response, instance, cases
                         except Exception as e:
                             # handle / log the error and reraise so the phone knows to resubmit
-                            _handle_unexpected_error(e, instance)
+                            # note that in the case of edit submissions this won't flag the previous
+                            # submission as having been edited. this is intentional, since we should treat
+                            # this use case as if the edit "failed"
+                            instance = _handle_unexpected_error(e, instance)
+                            instance.save()
```
Comment (czue) [inline corehq/ex-submodules/couchforms/util.py]: this just makes it more visually clear where the saves are happening

**Comment by czue** [comment]
(fixing build)

**Comment by czue** [comment]
updated, wait for tests

File: corehq/ex-submodules/couchforms/util.py (hunk 1 above)
Comment (dannyroberts) [inline corehq/ex-submodules/couchforms/util.py]: :+1:

**Comment by dannyroberts** [comment]
:+1:

**Pull request merged**